src/Microsoft.IIS.Administration.AccessManagement/Microsoft.IIS.Administration.AccessManagement.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Microsoft.IIS.Administration.AccessManagement</Description>
-    <VersionPrefix>1.1.0</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
@jimmyca15 I'm assuming all plugins version together? If so, I'd recommend pulling this out into a `common.props` file so you never end up accidentally forget to update the version on one of them.